### PR TITLE
V119-052: Add a code sample for custom environments

### PR DESCRIPTION
**Pull request by AnthonyLeonardoGracio**
This shows an example of VS Code workspace that modifies some environment variables at startup.

### CODE REVIEW

File: integration/vscode/Code Samples/custom_env/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Workspace-specific environment
+
+This folder contains a `custom_env.code-workspace` file that setups a custom
+environment for this workspace, setting and modifying the some environment
```
Comment (AdrienBoulanger) [inline integration/vscode/Code Samples/custom_env/README.md:4]: remove "the"

File: integration/vscode/Code Samples/custom_env/custom_env.code-workspace [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"ada.projectFile": "default.gpr",
+
+		// Set a workspace-specific environment
+		"terminal.integrated.env.linux": {
+			//  Set MAIN_NUMBER secnario variable to MAIN_2 directly from the environment
```
Comment (AdrienBoulanger) [inline integration/vscode/Code Samples/custom_env/custom_env.code-workspace:12]: (typo) scenario

File: integration/vscode/Code Samples/custom_env/default.gpr [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      when "MAIN_2" =>
+         for Main use ("main_2.adb");
+
+      when "MAIN_1" =>
+         for Main use ("main.adb");
+   end case;
+
+   --  Use the same executable name for main source files
+   package Builder is
+      for Executable ("main_1.adb") use "main";
```
Comment (AdrienBoulanger) [inline integration/vscode/Code Samples/custom_env/default.gpr:21]: main.adb?

Comment (AnthonyLeonardoGracio) [reply integration/vscode/Code Samples/custom_env/default.gpr:21]: Done (renamed file into main_1.adb)

File: integration/vscode/Code Samples/custom_env/custom_env.code-workspace (hunk 2 above)
Comment (AnthonyLeonardoGracio) [reply integration/vscode/Code Samples/custom_env/custom_env.code-workspace]: Done

File: integration/vscode/Code Samples/custom_env/README.md (hunk 1 above)
Comment (AnthonyLeonardoGracio) [reply integration/vscode/Code Samples/custom_env/README.md]: Done

**Review by AdrienBoulanger — APPROVED** [review]

File: integration/vscode/Code Samples/custom_env/custom_env.code-workspace [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+		},
+
+		// Set a workspace-specific environment for Windows
+		"terminal.integrated.env.windows": {
+			//  Set MAIN_NUMBER scenario variable to MAIN_2 directly from the environment
+			"MAIN_NUMBER": "MAIN_2",
+
+			//  Optionally, you can append the Ada toolchain of your choice to the PATH environment variable
+			//  if you want to use a different toolchain.
+			//  "PATH": "<PATH_TO_TOOLCHAIN>:${env:PATH}",
```
Comment (setton) [inline integration/vscode/Code Samples/custom_env/custom_env.code-workspace:31]: should we use `;` as path separator on Windows?

Comment (AnthonyLeonardoGracio) [reply integration/vscode/Code Samples/custom_env/custom_env.code-workspace:31]: it's commented code, but yes, I can change it

**Review by setton — APPROVED** [review]

**Pull request merged**